cmd/performance-profile-creator/cmd/root.go

```diff
@@ -277,6 +281,20 @@ func getClusterData(mustGatherDirPath string) (ClusterData, error) {
 	return cluster, nil
 }
 
+func validateMustGatherDirPath(path string) error {
```

Comment:
is there a test coverage for these scenarios?

Reply:
yes. we have unit tests that cover the complete flow